in_toto/model.go

```diff
@@ -112,6 +170,24 @@ type Step struct {
 	SupplyChainItem
 }
 
+func validateStep(step Step) error {
+	if step.Type != "step" {
+		return fmt.Errorf("invalid Type value for step: should be 'step'")
+	}
+	for _, keyId := range step.PubKeys {
+		if err := validateKeyId(keyId); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+/*
+ISO8601DateSchema defines the format string of a timestamp following the
+ISO 8601 standard.
+*/
+const ISO8601DateSchema = "2006-01-02T15:04:05Z"
```

Comment:
OT: Can you replace `RFC3339`  with `ISO8601DateSchema` in  `verifylib.VerifyLayoutExpiration`. I think it's okay to sneak this into this PR.